File: CPP Sept 2023/oop - 1/date.cpp
#include<iostream>
using namespace std;

class date{
    int day;
    int month;
    int year;
    public: 
    void  accept(){
        cout<<"enter day, month, year: ";
        cin>>day>>month>>year;

    }
    public:
    void display(){
        cout<<"Day: "<<day<<" Month: "<<month<<" Year: "<<year<<endl;
    }
    public:
    void setDate(int a, int b, int c){
        day = a;
        month = b;
        year = c;
    }
    public:
    void getDate(){
        cout<<"getData date is : ";
        cout<<day<<"/"<<month<<"/"<<year;
    }


};
int main(){
    date d;
    d.accept();
    d.display();
    d.setDate(1,2,3000);
    d.getDate();
}

File: Java Sept 2023/Inheritance/Assignment 4/src/service/Date.java
package service;
public class Date {
	private int dd;
	private int mm;
	private int yy;
	
	public Date() {}
	public Date(int d,int m, int y) {
		dd=d;
		mm=m;
		yy=y;
	}
	public void display() {
		System.out.println("Date display");
		System.out.println(dd+"/"+mm+"/"+yy);
	}

}

